pkg/security/podsecuritypolicy/provider.go

```diff
@@ -296,6 +296,28 @@ func (s *simpleProvider) ValidatePod(pod *api.Pod) field.ErrorList {
 							"Flexvolume driver is not allowed to be used"))
 				}
 			}
+
+			if fsType == policy.CSI {
+				// one or more drivers must be specified
+				if len(s.psp.Spec.AllowedCSIDrivers) == 0 {
+					allErrs = append(allErrs, field.Invalid(field.NewPath("spec", "allowedCSIDrivers"), s.psp.Spec.AllowedCSIDrivers,
```

Comment:
This is supposed to be validating the pod, not the PSP.  Refer to the flex driver error above as an example

Reply:
Got it.